app/controllers/ror_controller.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# A proxy to ROR for typeahead
+class RorController < ApplicationController
+  include Dry::Monads[:result]
+
+  def show
+    result = lookup(params.require(:q))
+    @suggestions = result.value_or([])
+    if result.success?
+      render status: :no_content, html: "" and return if @suggestions.empty?
+
+      render status: :ok, layout: false
+    else
+      logger.warn(result.failure)
```

Comment:
Probably not honeybadger-worthy, right?

Reply:
Probably not. (This was copied from FAST autosuggest, where we don't HB either.)